eventClass/eventHandler.py

```diff
@@ -1,5 +1,7 @@
 from collections import deque
-from aeroCubeEvent import AeroCubeEvent
+from .aeroCubeEvent import AeroCubeEvent, ResultEvent
+from .aeroCubeSignal import *
+from enum import Enum
 
 
 class EventHandler(object):

```

Comment:
Within the file itself, the functions should be re-ordered, the current order/groupings aren't very coherent.